README.md

```diff
@@ -1,45 +1,191 @@
+| Config     | Optional: OCI image specification for helm configuration and set flags                                        |
+| CRDs       | Optional: OCI image specification for additional CRDs that are pre-installed before helm charts are processed |
+
+```yaml
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Manifest
+metadata:
+  labels:
+    operator.kyma-project.io/channel: stable
+    operator.kyma-project.io/controller-name: manifest
+    operator.kyma-project.io/kyma-name: kyma-sample
+  name: manifestkyma-sample-delete
+  namespace: default
+spec:
+  remote: true
+  resource:
+    kind: SampleCRD
+    resource: samplecrds
+    apiVersion: operator.kyma-project.io/v1alpha1
+    metadata:
+      name: sample-crd-from-manifest
+      namespace: default
+    spec:
+      randomkey: samplevalue
+  crds:
+    ref: sha256:71cf4f1fee1a2f51296cc805aa9b24bc14fd5c2b4aee1e24aadc2996b067bb3d
+    name: kyma-project.io/module/example-module-name
+    repo: kcp-registry.localhost:8888/component-descriptors
+    type: oci-ref
+  config:
+    ref: sha256:61be4f1fee1a2f51296cc805aa9b24bc14fd5c2b4aee1e24aadc2996b067ccec
+    name: kyma-project.io/module/example-module-name
+    repo: kcp-registry.localhost:8888/component-descriptors
+    type: oci-ref
+  installs:
+    - source:
+        name: kyma-project.io/module/example-module-name
+        repo: kcp-registry.localhost:8888/component-descriptors
+        ref: sha256:c64f0580a74259712f24243528881a76b5e1c9cd254fa58197de93a6347f99b9
+        type: oci-ref
+      name: redis
 ```
-    manifestOperations := manifest.NewOperations(logger, r.RestConfig, cli.New())
-	if create {
-		if err := manifestOperations.Install("chart/path", releaseName, "repoName/chartName", repoName, url, args); err != nil {
-			return err
-		}
-	} else {
-		if err := manifestOperations.Uninstall("chart/path", "repoName/chartName", releaseName, args); err != nil {
-			return err
-		}
-	}
-  ```
-- `chartPath` takes priority over the **url** parameter. If the chart should be downloaded from `url` pass `chartPath` as empty string
-- `args` only supports `--set` and [action flags](https://github.com/helm/helm/blob/v3.9.0/pkg/action/install.go#L66), with variable names and values, comma-seperated. Example:
+
+If `.Spec.Remote.` is set to `true`, the operator will look for a secret with the name specified by Manifest CR's label `operator.kyma-project.io/kyma-name: kyma-sample`. 
+Follow steps in this [guide](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/creating-test-environment.md#install-kyma-and-run-lifecycle-manager-operator) to create the required secret.
+
+For more details on OCI Image **bundling** and **formats**, refer to our [bundling and installation guide](https://github.com/kyma-project/lifecycle-manager/tree/main/samples/template-operator#bundling-and-installation).
+The component-descriptor generated from this guide could be used for `Manifest Spec` OCI image specifications.
```

Comment:
```suggestion
The component-descriptor generated from this guide could be used to independantly build a `Manifest Spec` based on the OCI image specifications. 

>Note: In the Kyma Control-Plane, [Lifecycle-Manager](https://github.com/kyma-project/lifecycle-manager#how-it-works) translates these Layers automatically and provisions the CRs based on it's `ModuleTemplate` resource.
```

Reply:
`>Note: [Lifecycle-Manager](https://github.com/kyma-project/lifecycle-manager#how-it-works) translates these layers from a `ModuleTemplate` resource on the Kyma control-plane and translates them automatically to a subsequent `Manifest` resource.` ?